net/src/ipc_net_worker.rs

```diff
@@ -119,13 +149,91 @@ impl IpcNetWorker {
                 let out: Box<NetWorker> = Box::new(IpcClient::new(h, socket, block_connect)?);
                 Ok(out)
             }),
+            None,
         )
     }
 
     // -- private -- //
 
+    /// create a new IpcNetWorker instance pointing to a process that we spawn here
+    fn priv_spawn(
+        handler: NetHandler,
+        cmd: String,
+        args: Vec<String>,
+        work_dir: String,
+        env: HashMap<String, String>,
+        block_connect: bool,
+    ) -> NetResult<Self> {
+        let mut proc = std::process::Command::new(cmd);
+        proc.stdout(std::process::Stdio::piped());
+        proc.args(&args);
+        proc.envs(&env);
+        proc.current_dir(work_dir);
```

Comment:
`Command::new()` returns a struct that can be used in a "builder-like" manner. For example:
```rust
let proc_handler = Command::new(cmd)
    .stdout(std::process::Stdio::piped())
    .args(&args)
    .envs(&env)
    .current_dir(work_dir)
    .spawn()?;
```

Reply:
changed... although I still needed some stack space so I could print out the spawn command for debugging.